src/services/subject.js

```
@@ -0,0 +1,31 @@
+const Subject = require('~/models/subject')
+
+const subjectService = {
+  getSubjects: async () => {
+    return await Subject.find().lean().exec()
+  },
+  getSubjectById: async (id) => {
+    return await Subject.findById(id).lean().exec()
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)